src/main/groovy/au/org/ala/ecodata/metadata/OutputModelProcessor.groovy

```diff
@@ -194,6 +194,11 @@ class OutputModelProcessor {
         }
         boolean first = true
         Map toRepeat
+
+        // if datamodel contains flattenOption then set it
+        if (outputMetadata.metadata.flattenOption != null)
+            option = outputMetadata.metadata.flattenOption
```

Comment:
so the design here is to set a flag in the form template about how that particular output should be exported?
It would probably be clearer to promote this to the FormSection rather than the actual rendering template but i understand there is a time constraint here.